pandas/io/parsers.py

```diff
@@ -2276,7 +2276,11 @@ def __init__(self, f, **kwds):
 
         self.dtype = kwds["dtype"]
         self.thousands = kwds["thousands"]
-        self.decimal = kwds["decimal"]
+
+        if isinstance(kwds["decimal"], bytes):
```

Comment:
we don't accept bytes as parameters, they must be strings, this is just a mistake.

Reply:
oh in that case the fix is just the one in line 491? x)

Reply:
BTW, decimal is almost always bytes:

https://github.com/pandas-dev/pandas/blob/6d11aa89fc06ebc55519507f6652dcfcba35a302/pandas/io/parsers.py#L571

I agree it's a mistake though, that's why I opened #29653 (and we internally have a workaround where we patch pandas so we can proceed with our Python 3 migration).

Let me know if I should just go ahead and drop support for bytes.

Reply:
Yea just go ahead and remove this block